tensorflow/lite/kernels/internal/optimized/optimized_ops.h

```diff
@@ -265,7 +265,7 @@ inline void BinaryBroadcastFiveFold(const ArithmeticParams& unswitched_params,
       // We have broadcast y2*y3*y4 of input2 data y1 times, and now move on.
       input2_data_reset = input2_data_ptr;
     }
-  } else {
+  } else if (input1_data_ptr != nullptr) {
```

Comment:
Instead of adding nullptr checks here, how about adding the null pointer checking just after the input1_data_ptr declarations.

For example,

```
const T* input1_data_ptr = input1_data;
TFLITE_DCHECK(input1_data_ptr != nullptr);
const T* input2_data_reset = input2_data;
```

Reply:
please ignore the above comment.